src/server/system_services/schemas/nsfs_account_schema.js

```diff
@@ -50,6 +50,7 @@ module.exports = {
         },
         access_keys: {
             type: 'array',
+            maxItems: 2,
```

Comment:
I wouldn't put this in schema. We might want to have more deactivated items in the list.

Reply:
I will remove it.
I added it because I want to restrict the access_keys array from having more than 2 items, in what case do you think would we want to have more than 2 objects?

Reply:
It's a superficial limit. we can limit it in the api if needed, but overall there shouldn't be anything that prevents us from increasing that limit in the api and everything should work fine.